appshell/cefclient_mac.mm

```diff
@@ -334,8 +334,15 @@ - (void)windowWillEnterFullScreen:(NSNotification *)notification {
     }
     if (customTitlebar) {
         [customTitlebar setHidden:YES];
+        NSWindow *window = [notification object];
+
+        // Since we have nuked the title, we will have
+        // to set the string back as we are hiding the
+        // custom title bar.
+        [window setTitle:[customTitlebar titleString]];
```

Comment:
This was happening earlier with 10.11 sdk. Please refer to my comment https://github.com/adobe/brackets-shell/pull/544/files/d3cd41c8fb964cd5634b3babdeec9dd25e681861#r63060349

for more explanation on this,